### DISCUSSION TITLE
BC400 / hc1 mqtt discovery current room temperature

### DISCUSSION THREAD

**Discussion by Delta1977** [emsesp/EMS-ESP32#2004] [Q&A] [answered]
In BC400 is a entity called "hc1 mqtt discovery current room temperature" with value roomTemp

What cannot be done with it ?

**Comment by proddy** [comment, marked as answer]
it's used to tell Home Assistant which temperature to show in the climate control, as in previous versions there could only be one value. Some people want to HA to show the value from an external sensor, instead of using what the thermostat is showing.
